src/main/java/core/basesyntax/service/WritingIntoCsvFileService.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.service;
+
+import java.util.List;
+
+public interface WritingIntoCsvFileService {

```

Comment:
It's better to rename this interface and make it more abstract (not using Csv) like FileWriterService for example